### ISSUE TITLE
[Bug Report] Dialog/Menu/Tooltip output an extra element and break layout

### ISSUE THREAD

**Issue by augustoborn** [vuetifyjs/vuetify#6951]
### Versions and Environment
**Vuetify:** 1.5.9
**Vue:** 2.6.10
**Browsers:** Chrome 73.0.3683.103
**OS:** Linux x86_64

### Steps to reproduce
Add a dialog inside a nav drawer, using a v-list-tile in the activator slot of the dialog.

### Expected Behavior
The dialog activator should have the exact spacing as the other drawer items.

### Actual Behavior
The dialog activator has extra spacing below it.

### Reproduction Link
<a href="https://codepen.io/anon/pen/ROKPZo" target="_blank">https://codepen.io/anon/pen/ROKPZo</a>

### Other comments
Poking around the inspector, it looks like the problem stems from the v-dialog__container div right under the v-list-tile activator. Setting its style to display: none fixes the issue.

**Comment by blalan05** [comment]
Possible solution: There is an "extra" div added after the first v-list-tile with the inline style of display: inline-block; changing it to display: none fixes the styling.

**Comment by usb248** [comment]
Any fix ?

**Comment by augustoborn** [comment]
> Any fix ?

My workaround was using this in my app's main CSS file:

.v-dialog__container {
  display: none !important;
}

**Comment by ipluser** [comment]
how to fix it?

**Comment by matthewma7** [comment]
The question here is why it needs to be inline-block? https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VDialog/VDialog.js#L331

**Comment by usb248** [comment]
> 
> 
> > Any fix ?
> 
> My workaround was using this in my app's main CSS file:
> 
> .v-dialog__container {
> display: none !important;
> }

@augustoborn  very dirty ...

**Comment by gorsat** [comment]
Same problem in v-menu with the activator slot. The same dirty fix works: 
.v-menu--inline {
  display: none !important;
}
1. Is there a real fix coming? 
2. Should I open a different bug for this one?

**Comment by jessielaf** [comment]
Is there a fix coming this or is this intended behavior?

**Comment by johnleider** [comment]
I'm working on a fix for this right now. In the meantime, here is a workaround: https://codepen.io/johnjleider/pen/oNvpXrv

**Comment by xyannik** [comment]
Hi, I posted [this issue](https://github.com/vuetifyjs/vuetify/issues/8777) which was marked as duplicate of 6951. But I didn't get a real response on my question: Is it expected behaviour, that in 2.0 dialogs/menus/tooltips in the HTML get added as a sibling of the activator? In 1.5 they got added as child-elements instead of siblings. Since I often use them in Grid-/Flex-Layouts, this has caused a couple of render issues. I'd like to know if this will be reverted in the next release, before I change all my related CSS (either way is fine, I just don't want to change all my CSS, only to have to change it back in the next release). Thanks
